src/pages/Authentication/SignUp/SignUp.js

```diff
@@ -1,9 +1,17 @@
 import React from "react";
+import "./signUp.css";
+import { Header, SignUpCard, Footer } from "../../../components/index";
 
 const SignUpPage = () => {
     return (
-        <div>
-            SignUpPage
+        <div className="ap_signup-page">
+            <div className="ap_signup-page-header-section">
+                <Header />
```

Comment:
you can use only once Header and Footer component in your app.js where you have written all routes (pages). no need to call in each and every page

Reply:
but wouldn't that create issues!!? Like i wanted my header to be fixed to top therefore i have gave it a different styling in home page, and in productlisting page it's styling was a little different!!??
Do lemme know your opinions on this front!!??
And if that's something posible importing only once in app.js like you suggested than it will be super awesome!! Will reduce a lot of my work!!